### PR TITLE
Update SymEngine.py Dev Binaries

### PR DESCRIPTION
**Pull request by ShikharJ**

### CODE REVIEW

**Comment by certik** [comment]
@ShikharJ any ideas why it failed? Looking at the log (https://travis-ci.org/symengine/python-symengine-feedstock/jobs/266783411) it looks like a bug somewhere else than symengine?

**Comment by isuruf** [comment]
@ShikharJ, I changed the branch. (It should be the tagged branch) Can you fix the conflicts?

**Comment by ShikharJ** [comment]
@isuruf We've just updated the `SymEngine` binaries, still this is unable to access `symengine/solve.h`. Why is that?

**Comment by isuruf** [comment]
You have to change Line 22 and Line 28

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% set commit = "b409536eb9029b274d2956ec14a7954a0089aa3a" %}
-{% set version = "0.3.0" %}
+{% set version = "0.2.0.213.g296fce14" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: This is wrong. Do `git fetch upstream_remote_name --tags` to fetch the latest tags

**Comment by ShikharJ** [comment]
Ping @isuruf.

**Pull request merged**